docs/user_guide/3_installing_the_software.rst

```diff
@@ -0,0 +1,83 @@
+Installing the software on a new PC
+====================================
+
+.. note:: By default, we will provide machines that already have all the software set up for you.
+However, even though each delivery will consist of a NUC-CONTROL machine for Hand's driver (which we always recommend to use), the SERVER Laptop is optional.
+In case you want to set up a custom machine as a SERVER, please follow the instructions below.
+The values for each field can be found in the ``Hand Delivery Instructions`` provided with the hand.
+
+Hardware specifications
+-----------------------
+
+In order to run our software and its dependencies you will need to meet some hardware requirements.
+
++---------------+------------------------------------------------------------------------------------------------------------+
+| CPU           | Intel i5 or above (i7 recommended)                                                                         |
++---------------+------------------------------------------------------------------------------------------------------------+
+| RAM           | 4GB or above (16 GB recommended)                                                                           |
++---------------+------------------------------------------------------------------------------------------------------------+
+| Hard Drive    | Fast HDD or SSD as laptop HDD are very slow (500 GB SSD recommended)                                       |
++---------------+------------------------------------------------------------------------------------------------------------+
+| Graphics Card | Nvidia GPU (optional)                                                                                      |
++---------------+------------------------------------------------------------------------------------------------------------+
+| LAN           | A spare LAN port to connect the Hand (even with a USB to LAN adaptor)                                      |
++---------------+------------------------------------------------------------------------------------------------------------+
+| OS            | Ubuntu 20.04 Noetic (Active development), 18.04 Melodic, 16.04 Kinetic or 14.04 Indigo for older releases. |
++---------------+------------------------------------------------------------------------------------------------------------+
+
+The most important one is to have a fast HDD or an SSD.
+
+We have created a one-liner that is able to install Docker, download the docker image and create a new container for you.
```

Comment:
Inconsistent capitalisation of `Docker`. 2 versions used even in this single sentence. Or is it deliberate that Docker as a tool is proper noun but docker type image is not?

Reply:
fixed, leaved all with capital